src/gafaelfawr/handlers/auth.py

```diff
@@ -101,6 +122,39 @@ async def get_auth(request: web.Request, token: VerifiedToken) -> web.Response:
     if satisfy not in ("any", "all"):
         msg = "satisfy parameter must be any or all"
         raise web.HTTPBadRequest(reason=msg, text=msg)
+    auth_type_name = request.query.get("auth_type", "bearer")
+    if auth_type_name not in ("basic", "bearer"):
+        msg = "auth_type parameter must be basic or bearer"
+        raise web.HTTPBadRequest(reason=msg, text=msg)
```

Comment:
It would -- I'm not sure where else to send it, though. At least it gives people a good error to cut and paste. And yeah, it indicates a misconfiguration of the ingress-nginx annotations.